build.gradle.kts

```diff
@@ -54,6 +57,20 @@ tasks.publishPlugin {
     channels.set(listOf(project.version.toString().split('-').getOrElse(1) { "default" }.split('.').first()))
 }
 
+tasks {
+    withType<ShadowJar> {
+        archiveBaseName = "detekt-intellij-plugin-shaded"
+        mergeServiceFiles()
+        relocate("org.jetbrains.kotlin", "detekt.shadow.org.jetbrains.kotlin")
+    }
+}
+
+tasks.withType<PrepareSandboxTask> {
+    pluginJar = project.tasks.withType<ShadowJar>().getByName("shadowJar").archiveFile
+    // disable to collect libraries located in runtime classpath (replace with empty file collection)
+    runtimeClasspathFiles = project.objects.fileCollection()
```

Comment:
https://github.com/JetBrains/gradle-intellij-plugin/blob/a10fd139467dd34c8e292296b563edfed14fea9d/src/main/kotlin/org/jetbrains/intellij/IntelliJPlugin.kt#L611
`runtimeClasspathFiles` uses to include depended libraries and plugins. 
From this changes applied, we use shadow jar, which included libraries, so we can replace `runtimeClasspathFiles` empty.